phpunit.xml

```diff
@@ -1,12 +1,12 @@
 <phpunit>
   <php>
-    <env name="RESOURCEKEY" value="!!YOUR_RESOURCE_KEY!!" force="true"/>
-    <env name="FOD_CLOUD_API_URL" value="" force="true"/>
+    <env name="RESOURCEKEY" value="!!YOUR_RESOURCE_KEY!!"/>
+    <env name="FOD_CLOUD_API_URL" value=""/>
   </php>
   <testsuites>
-    <testsuite name="Tests">
-      <file>examples/index.php</file>
-      <file>tests/geolocation.php</file>
+    <testsuite name="Integration">
+      <file>tests/ExampleTests.php</file>
+      <file>tests/GeoLocationTests.php</file>
```

Comment:
Don't `GeoLocationTests` fall into the `Unit` category?

Reply:
Not sure, but they require a resource key.

Reply:
Not sure, but they require a resource key.

Reply:
Fair enough. I suppose if they're calling the actual service, then you're correct, they are integration.